src/Bindings/PluginManager.cpp

```diff
@@ -78,9 +77,9 @@ void cPluginManager::RefreshPluginList(void)
 	// Set all plugins with invalid folders as psNotFound:
 	for (auto & plugin: m_Plugins)
 	{
-		if (std::find(Folders.cbegin(), Folders.cend(), plugin->GetFolderName()) == Folders.end())
+		if (std::find(Folders.cbegin(), Folders.cend(), plugin->GetLocalFolder()) == Folders.end())
```

Comment:
Wrong, `GetLocalFolder` returns the path including the "Plugins/", `Folders` never contains the "Plugins/" prefix.
You need to call `GetFolderName`